apps/web/src/store/modules/settings/type.ts

```diff
@@ -7,5 +7,5 @@ export interface SetItemRequest {
 export type SettingItem = Record<string, any>;
 
 export interface SettingsState {
-    items: SettingItem;
+    user: SettingItem;
 }
```

Comment:
suggestion: what about to make it flat?

```suggestion
export type SettingsState = SettingItem;
```